mempool/src/runtime.rs

```diff
@@ -28,7 +33,11 @@ impl MempoolRuntime {
     pub fn bootstrap(
         config: &NodeConfig,
         network_sender: MempoolNetworkSender,
-        network_events: MempoolNetworkEvents,
+        network_events: Vec<MempoolNetworkEvents>,
+        ac_endpoint_listener: Receiver<(
```

Comment:
nit: we use different naming for this structure in different places. Can we unify it?